doc/csclng.adoc

```diff
@@ -24,12 +24,20 @@ The following parameters are given to clang by default:
 
     * -Xanalyzer -analyzer-output=text
 
+    * -fno-caret-diagnostics
+
 The following parameters are passed to clang from compiler's command line:
 
     * -D...
 
     * -I...
 
+    * -O...
+
+    * -fexceptions
+
+    * -fno-exceptions
```

Comment:
That would prevent `clang --analyzer` from running in too many cases because many `-f...` flags supported by gcc throw a fatal error when they are passed to `clang`.